src/components/CodeBlock/index.tsx

```diff
@@ -26,6 +28,9 @@ export const CodeBlock = (props: CodeBlockProps) => {
     const matches = className.match(/language-(?<lang>.*)/)
     const [code, setCode] = useState('')
     const [projectName, setProjectName] = useState('')
+    const [hasBeenCopied, setHasBeenCopied] = useState(false)
+    const [spin, setSpin] = useState(false)
+    const [tooltipVisible, setTooltipVisible] = useState(false)
```

Comment:
😄
IMO the animation is excessive, especially since the tooltip shows up when the rotation finishes. So I'd leave just `hasBeenCopied`.

Reply:
Yeah I guess it's a bit long too

Reply:
will remove